docs/install/local-docker.md

```diff
@@ -0,0 +1,83 @@
+# Install MLRun locally using Docker
+
+You can install and use MLRun and Nuclio locally on your computer, this will not include all the services and elastic 
+scaling capabilities which you can get with the Kubernetes based deployment but is much simpler to start with.
+
+```{admonition} Note
+- Using Docker is limited to local, Nuclio, and Serving runtimes and local pipelines.
+```
+
+The installation is done using [`docker compose`](https://docs.docker.com/compose/) and will deploy MLRun service, 
+MLRun UI, Nuclio serverless engine, and optionally Jupyter server. 
+
+There are two installation options:
+- [**Use MLRun with your own client (PyCharm, VSCode, Jupyter)**](#use-mlrun-with-your-own-client)
+- [**Use MLRun with MLRun Jupyter image (pre loaded with examples/demos)**](#use-mlrun-with-mlrun-jupyter-image)
+
+In both cased you need to set the `SHARED_DIR` environment variable to point to a host path for storing MLRun artifacts and DB, 
+for example `export SHARED_DIR=~/mlrun-data` (or use `set SHARED_DIR=c:\mlrun-data` in windows), make sure the directory exist.
+
+It is recommended to set the `HOST_IP` variable with your computer IP address (required for Nuclio dashboard), 
+you can check your IP address using the `ip addr` command (or `ipconfig` in windows).
```

Comment:
It is recommended to set the `HOST_IP` variable with your computer IP address (required for Nuclio dashboard). You can check your IP address using the `ip addr` command (or `ipconfig` in windows).
>>>2 sentences